Azure/sonic-buildimage
# [centec] support v682-48y8c and v682-48x8c

#### Why I did it
Adding platform support for centec v682-48y8c and v682-48x8c.
V682-48y8c switch has 48 SFP+ (1G/10G/25G) ports, 8 QSFP28 (40G/100G) ports on CENTEC TsingMa.MX.
V682-48y8c is different from V682-48y8c_d in that:
1. transceiver is managed by cpu smbus rather than TsingMa.MX i2c bus.
2. port led is managed by mcu inside TsingMa.MX.
3. fan, psu, sensors, leds are managed by cpu smbus other than the cpu board vendor's close sourse driver. 

V682-48y8c switch has 48 SFP+ (1G/10G) ports, 8 QSFP28 (40G/100G) ports on CENTEC TsingMa.MX.
CPU used in v682-48y8c and v682-48x8c is Intel(R) Xeon(R) CPU D-1527.

#### How I did it
1. Modify related code in platform and device directory.
2. Upgrade centec sai to v1.9.
3. upgrade python to python3 and kernel version to 5.0 for V682-48y8c_d.

#### How to verify it
Build centec amd64 sonic image, verify platform functions (port, sfp, led etc) on centec v682-48y8c and v682-48x8c  board.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
